src/Forms/FormRequestHandler.php

```diff
@@ -103,7 +103,7 @@ public function Link($action = null)
      * if the form is valid.
      *
      * @param HTTPRequest $request
-     * @return HTTPResponse
+     * @return mixed
      * @throws HTTPResponse_Exception
      */
     public function httpSubmission($request)
```

Comment:
Should this be given explicit typehints? I think `: mixed` is better than a `@return mixed` phpdoc

Reply:
I could be, however at that point I'm just doing general refactoring, which I could do pretty much forever. I need to set boundaries for what I change on this PR.

I changed the phpdoc because it's simply wrong and confusing if you start debugging things